### PR TITLE
Release vision 0.35.1

### PR DESCRIPTION
**Pull request by crwilcox**
This pull request was generated using releasetool.

### CODE REVIEW

File: vision/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 
+## 0.35.1
+
+11-21-2018 11:25 PST
+
+### Implementation Changes
+- Add ProductSearchClient to vision.py ([#6635](https://github.com/googleapis/google-cloud-python/pull/6635))
+
+### Internal / Testing Changes
+- Vision: pick up fixes to GAPIC generator. ([#6632](https://github.com/googleapis/google-cloud-python/pull/6632))
```
Comment (tseaver) [inline vision/CHANGELOG.md]:
```suggestion
- Pick up fixes to GAPIC generator. ([#6632](https://github.com/googleapis/google-cloud-python/pull/6632))
```

Comment (crwilcox) [reply vision/CHANGELOG.md]: We were frozen and so I was holding this :)

**Comment by tseaver** [comment]
@crwilcox Have you run `releasetool tag` yet?  I don't see the release on PyPI, or on the [releases page](https://github.com/googleapis/google-cloud-python/releases).

**Comment by crwilcox** [comment]
Release is at https://github.com/googleapis/google-cloud-python/releases/tag/vision-0.35.1

**Pull request merged**